Library/Martial Arts/Martial Arts Skills.skl

```diff
@@ -1884,6 +1796,27 @@
 				"name": "@Melee Weapon Skill@",
 				"modifier": -5
 			},
+			"limit": 0
+		},
+		{
+			"id": "a2df4d38-06b2-42a9-b529-d80b969dd90e",
+			"type": "technique",
+			"name": "Horse Archery",
```

Comment:
Horse Archery does have a Riding prerequisite.

Reply:
I did include this.
Line 1830